app/src/test/java/ch/epfl/sweng/erpa/operations/RemoteServicesProviderCoordinatorTest.java

```diff
@@ -158,6 +159,12 @@ public void terminate() {
         fuse.ignite();
     }
 
+    @Override
+    public GameService getGameService()
```

Comment:
Why the RSP coordinator is aware of the game service???

Reply:
Because otherwise I cannot compile, due to the fact that SyntheticRSP extends RSP, and that RSP needs a getGameService method